controllers/suggestions/suggest_match_video_review_controller.py

```diff
@@ -37,12 +39,25 @@ def get(self):
         self.response.out.write(jinja2_engine.render('suggestions/suggest_match_video_review_list.html', self.template_values))
 
     def post(self):
-        accept_keys = map(lambda x: int(x) if x.isdigit() else x, self.request.POST.getall("accept_keys[]"))
-        reject_keys = map(lambda x: int(x) if x.isdigit() else x, self.request.POST.getall("reject_keys[]"))
-
+        accept_keys = []
+        reject_keys = []
+        for value in self.request.POST.values():
+            logging.debug(value)
```

Comment:
Remove and remove import.

Reply:
Alright. Want me to open another PR to remove them from other suggestion controllers?